doc/source/ray-core/doc_code/obj_ref.py

```diff
@@ -0,0 +1,16 @@
+import ray
+
+@ray.remote
+def echo_and_get(x):  # List[ObjectRef]
+    """This function prints its input values to stdout."""
+    print("args:", x)
+    print("values:", ray.get(x))
+
+# Put three values into Ray's object store.
+a, b, c = ray.put(1), ray.put(2), ray.put(3)
+
+# Passing an object as a nested argument to `echo_and_get`. Ray does not
+# de-reference nested args, so `echo_and_get` sees the references.
+echo_and_get.remote([a, b, c])
```

Comment:
For symmetry, it would be nice if the other example also uses (a, b, c) as its arguments.

Could also add a Tip to use *args in python (I've seen Ray users ask about how to do this before).